nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -316,14 +316,34 @@ in
           '';
         };
 
+    virtualisation.bootPartition =
```

Comment:
Was the legacy boot broken before or broken by your changes?

Reply:
Broken by my changes